idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -1,6 +1,23 @@
+enum State {
+    REFUSING = 0,
+    ACCEPTING = 1,
+    STOPPING = 2,
+    STOPPED = 3,
+}
+
+enum RequestType {
+    LEGACY = 0,
+    TRAFFIC = 1,
+}
+
+struct HealthRequest {
+    1: optional RequestType type
```

Comment:
What about HealthRequestType? In case we ever add another function which has different request types.